tests/runtestcase

```diff
@@ -57,14 +57,16 @@ find_cores() {
     # for all machines in cluster 
     local COREAGE=60
 
-    local cpat=$(< /proc/sys/kernel/core_pattern)  # like cat core_pattern
-    local COREDIR=`dirname $cpat`; 
+    local CPAT=$(< /proc/sys/kernel/core_pattern)  # like cat core_pattern
```

Comment:
Could you please remove irrelevant changes?

Reply:
They are irrelevant to the cleanup, but relevant otherwise.

Reply:
Sorry not following. Why is it relevant?